src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/EventLogsPipelineUnitTests.cs

```diff
@@ -263,6 +263,18 @@ private static void ValidateLoggerRemoteCategoryInformationMessage(StreamReader
             Assert.Equal(string.Empty, result.EventName);
             Validate(result.Scopes, ("BoolValue", "true"), ("StringValue", "test"), ("IntValue", "5"));
             Validate(result.Arguments, ("Arg", "6"));
+
+            message = reader.ReadLine();
+            Assert.NotNull(message);
+
+            result = JsonSerializer.Deserialize<LoggerTestResult>(message);
```

Comment:
Note this test is disabled in the default configuration due to existing flakiness, but I was able to run it successfully locally.